static/src/js/components/CollectionResults/CollectionResultsHeader.js

```diff
@@ -4,8 +4,51 @@ import { Col, Form, Row } from 'react-bootstrap'
 
 import './CollectionResultsHeader.scss'
 
-const CollectionResultsHeader = ({ portal }) => {
+const eosdisTagKey = 'gov.nasa.eosdis'
```

Comment:
Probably want to move this into the application config in static config -- or somewhere not hard coded in a file anyway.